### DISCUSSION TITLE
Modificar campo de cadastro do cliente

### DISCUSSION THREAD

**Discussion by tsg-tiago** [RamonSilva20/mapos#2109] [Ideas]
Boa tarde pessoal, gostei muito do programa, possuo uma oficina mecanica e ele da conta do recado!! Facil d mexer, objetivo e nao dependo de internet pra funcionar!!
Mas eu precisava da ajuda de algum amigo pra me ensinar ou modificar algum campo do cadastro do cliente, pra colocar a placa do carro, seria possivel? Não precisa nem ser o modelo, só a placa mesmo
Tipo trocar o numero do CPF pela placa do veiculo
Desde ja muito obrigado a todos!!

**Comment by Felipebzzz** [comment]
Boa Noite,  conseguiu resolver?

**Comment by tsg-tiago** [comment]
Boa noite, consegui nada, coloco a placa do carro junto do nome do cliente, tá funcionando...

Enviado do Yahoo Mail no Android 
 
  Em dom, 25 25e jun 25e 2023 às 20:12, ***@***.***> escreveu:   


Boa Noite, conseguiu resolver?

**Reply by Felipebzzz** [reply]
Entre em contato comigo pelo e-mail, caso queira resolver este problema 

felipebarbosa1996@hotmail.com.br
